packages/components/src/components/input-field/input-field.jsx

```diff
@@ -139,18 +140,18 @@ const InputField = ({
 
             increment_value = parseFloat(getClampedValue(new_value)).toFixed(decimal_places);
         } else if (is_crypto || (!currency && is_float)) {
-            const new_value = parseFloat(+(current_value || 0)) + parseFloat(1 * 10 ** (0 - decimal_places));
-            increment_value = parseFloat(new_value).toFixed(decimal_places);
+            const new_value =
+                parseFloat(+(current_value || 0)) +
+                parseFloat(1 * 10 ** (0 - (decimal_point_change || decimal_places)));
```

Comment:
what is `decimal_point_change` and what does it do?

Reply:
The `decimal_point_change` property will specify which decimal point must be updated when the increment/decrement button is pressed
Eg - I have a digit 12.03, when I press `+` button, value changes for 12.04

Reply:
Please add a comment in the code to explain about the prop

Reply:
Will add the comment explaining the prop